### PR TITLE
feat(xkcd): refresh cron and `n + 1` entry fetching

### PR DESCRIPTION
**Pull request by kyranet**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,18 +37,22 @@
...
+		"tslib": "^2.5.3",
+		"turndown": "^7.1.2"
 	},
 	"devDependencies": {
 		"@commitlint/cli": "^17.6.5",
 		"@commitlint/config-conventional": "^17.6.5",
 		"@sapphire/eslint-config": "^4.4.2",
 		"@sapphire/prettier-config": "^1.4.5",
 		"@sapphire/ts-config": "^4.0.0",
+		"@types/cron": "^2",
```
Comment (favna) [inline package.json:52]: full semver (just use `yarn upgrade-interactive`)

File: src/lib/setup/refresh-xkcd.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { refreshComicsFromRemote } from '#lib/utilities/xkcd';
+import { job } from 'cron';
+
+export const xkcdRefresh = job({
+	// cronTime: '0 0 12,13 * * 1-5',
```
Comment (favna) [inline src/lib/setup/refresh-xkcd.ts:5]: comment

File: src/lib/setup/refresh-xkcd.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+	cronTime: '0 35 14 * * 1-5',
+	onTick: refreshComicsFromRemote,
+	start: true,
+	unrefTimeout: true,
+	timeZone: 'Etc/UTC'
```
Comment (favna) [inline src/lib/setup/refresh-xkcd.ts:10]: ArchId server has UTC+2 (CEST) (`Europe/Berlin`)

**Review by favna — CHANGES_REQUESTED** [review]

File: src/lib/setup/refresh-xkcd.ts (hunk 3 above)
Comment (kyranet) [reply src/lib/setup/refresh-xkcd.ts:10]: The GH action runs on UTC.

**Review by favna — APPROVED** [review]

**Pull request merged**